scripts/timelapse-colorizer-data/convert_emt_migration_data.py

```diff
@@ -142,21 +154,23 @@ def make_dataset(
     reduced_dataset[INITIAL_INDEX_COLUMN] = reduced_dataset.index.values
     grouped_frames = reduced_dataset.groupby(TIMES_COLUMN)
 
+    # Get the units and human-readable label for each feature; we include this as
+    # metadata inside the dataset manifest.
+    feature_labels = []
+    feature_metadata: List[FeatureMetadata] = []
+    for i in range(len(FEATURE_COLUMNS)):
+        (label, unit) = extract_units_from_feature_name(FEATURE_COLUMNS[i])
```

Comment:
nit: since we're just iterating through a list, this could just be
```
for feature in FEATURE_COLUMNS:
    (label, unit) = extract_units_from_feature_name(feature)
    ...
```
which feels slightly more pythonic to me